quickwit-metastore/src/split_metadata_version.rs

```diff
@@ -96,6 +97,10 @@ pub(crate) struct SplitMetadataV1 {
     /// in the storage URI resolver: for instance, the Amazon S3 region.
     pub split_id: String,
 
+    /// Id of the index this split belongs to.
+    #[serde(default)]
+    pub index_id: String,
```

Comment:
1. We could put "unknown" instead.
2. All the old split metadata deserialized from postgresql or the filebacked metastore will have "unknown" index id. This is a big issue.

One solution could be to override this value when we load the index in the file backed index and when we build the split in the postgresql metastore.

Reply:
> One solution could be to override this value when we load the index in the file backed index and when we build the split in the postgresql metastore.

Yes.